app/components/liquidity/addLiquidity.js

```diff
@@ -557,16 +557,31 @@ const AddLiquidity = ({
       </ModalContent>
     </Modal>
 
-    <SpinModal
-      isOpenModal={isOpenModal}
-      onCloseModal={onCloseModal}
-      title="Success"
-    >
-      <Box textAlign="center" mt={3} mb={8}>
-        Providing {fromValue} {fromSelectedToken.symbol} and {toValue}{' '}
-        {toSelectedToken.symbol}
-      </Box>
-    </SpinModal>
+    <Modal isOpen={isOpenModal} onClose={onCloseModal} isCentered>
+      <ModalOverlay />
+      <ModalContent bg="#120136" color="#fff" borderRadius="20px" width="90%">
+        <ModalCloseButton
+          bg="none"
+          border="0px"
+          color="#fff"
+          cursor="pointer"
+          _focus={{ outline: 'none' }}
+          onClick={onCloseModal}
+        />
+        <ModalBody align="center" my={2}>
+          <Circle size="70px" background="#68C18A" my={3}>
+            <CheckIcon fontSize="40px" />
+          </Circle>
+          <Text fontSize="18px" fontWeight="normal">
+            Transaction Succesful
+          </Text>
+          <Box textAlign="center" mt={3} mb={8}>
+            you have successfully Provided Liquidity for {fromValue}{' '}
+            {fromSelectedToken.symbol} and {toValue} {toSelectedToken.symbol}
```

Comment:
The values are supposed to be from the blockchain because the amount of any of the two tokens provided as liquidity may change by the time the transaction is mined.

Reply:
i have implemented this by getting the data from blockchain